README.md

```diff
@@ -630,3 +631,24 @@ One of three out comes can occur after the export retried has completed.
 
 > *NOTE: Changing Writable.Pipeline.ExecutionOrder will invalidate all currently stored data and result in it all being removed from the database on the next retry.* This is because the position of the export function can no longer be guaranteed and no way to ensure it is properly executed on the retry.
 
+### Secrets
+
+Application Services can retrieve their secrets from the underlying secret store using the GetSecrets() API in the SDK. When security is disabled, the secrets are retrieved from the Writable.InsecureSecrets section of the configuration file. An example of the configuration for insecure secrets is below.
```

Comment:
Should mention something about vault when in secure mode and a placeholder on how to add credentials to vault, i.e. the new story.